maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -107,8 +113,10 @@ fun Marker(
     onInfoWindowClick: (Marker) -> Unit = {},
     onInfoWindowClose: (Marker) -> Unit = {},
     onInfoWindowLongClick: (Marker) -> Unit = {},
+    infoWindowComponent: ComposeInfoWindowComponent? = null,
```

Comment:
Why a new type for this instead of `content: @Composable (Marker) -> Unit`?

Reply:
This should use the composable function type and not a new type to differentiate overloads on. Use an alternative name for the `Marker` function to differentiate these overloads instead.